cmdeploy/src/cmdeploy/__init__.py

```diff
@@ -479,6 +479,38 @@ def deploy_mtail(config):
     )
 
 
+def deploy_iroh_relay() -> None:
+    server.shell(
+        name="Download iroh-relay",
+        commands=[
+            "(echo '8af7f6d29d17476ce5c3053c3161db5793cb2ac49057d0bcaf689436cdccbeab  /usr/local/bin/iroh-relay' | sha256sum -c) || curl -L https://github.com/n0-computer/iroh/releases/download/v0.27.0/iroh-relay-v0.27.0-x86_64-unknown-linux-musl.tar.gz | gunzip | tar -x -f - ./iroh-relay -O >/usr/local/bin/iroh-relay",
```

Comment:
what is the checksum check really doing?  the check does not happen after the file is downloaded -- so we are basically running an unchecked binary?